cli/analyze/analyze.go

```diff
@@ -31,6 +32,8 @@ const (
 var (
 	flags = flag.NewFlagSet("analyze", flag.ContinueOnError)
 
+	keepGoingFlag = flags.Bool("keep_going", false, "Continue querying the dependency graph even after encountering errors.")
```

Comment:
I could see the case for making this the default. Not all repos will support a wildcard unconfigured query and the warning shown if it fails is sufficient feedback.

Reply:
Done